nodebuilder/node.go

```diff
@@ -27,9 +30,12 @@ import (
 	"github.com/celestiaorg/celestia-node/nodebuilder/state"
 )
 
-const Timeout = time.Second * 15
+var Timeout = time.Second * 30
```

Comment:
The issue is that this number will actually grow with the number of blocks in the DAGstore. It takes quite a while to start already. Maybe we can parallelize this process upstream, or find some other way to let the dagstore initialize before its used by the rest of node

Reply:
Thus the issue to make it configurable. Also, keep in mind that it took long on the the node that got kernel panic and had a corrupted DAGStore state